src/scripts/modules/media/body/embeddables/exercise-template.html

```diff
@@ -1,3 +1,6 @@
+{{#if data.items.[0]stimulus_html}}
```

Comment:
Jenkins is failing from a syntax error here
```suggestion
{{#if data.items.0.stimulus_html }}
```